ui/src/app/views/account-creation-modals/component.spec.ts

```diff
@@ -0,0 +1,71 @@
+import {DebugElement} from '@angular/core';
+import {ComponentFixture, fakeAsync, TestBed, tick} from '@angular/core/testing';
+import {FormsModule} from '@angular/forms';
+import {By} from '@angular/platform-browser';
+import {RouterTestingModule} from '@angular/router/testing';
+
+import {ClarityModule} from '@clr/angular';
+
+import {ProfileService} from 'generated';
+
+import {ProfileServiceStub} from 'testing/stubs/profile-service-stub';
+import {ServerConfigServiceStub} from 'testing/stubs/server-config-service-stub';
+
+import {
+  updateAndTick
+} from '../../../testing/test-helpers';
+import {ServerConfigService} from '../../services/server-config.service';
+
+import {AccountCreationModalsComponent} from '../account-creation-modals/component';
+import {RoutingSpinnerComponent} from '../routing-spinner/component';
+
+class AccountCreationModalsPage {
+  fixture: ComponentFixture<AccountCreationModalsComponent>;
+  component: AccountCreationModalsComponent;
+
+  constructor(testBed: typeof TestBed) {
+    this.fixture = testBed.createComponent(AccountCreationModalsComponent);
+    this.component = this.fixture.componentInstance;
+    this.readPageData();
+  }
+
+  readPageData() {
+    updateAndTick(this.fixture);
+  }
+}
+
+
+describe('AccountCreationModalsComponent', () => {
+  let profileServiceStub: ProfileServiceStub;
```

Comment:
I would suggest that we only need this variable if we are in fact testing some functionality via the stub. If we're simply using it to provide a value (and not verifying anything), you can just inline the new ProfileServiceStub in the provider definition.

Reply:
👍 Will do